src/components/reservation/ReservationModalBody.tsx

```diff
@@ -34,24 +30,27 @@ import {
   useGetPatronInformationByPatronIdV2
 } from "../../core/fbs/fbs";
 import { Manifestation } from "../../core/utils/types/entities";
+import { constructReservationData, getAuthorLine } from "./helper";
+import UseAvailableManifestations from "../../core/utils/UseAvailableManifestations";
 
 export const reservationModalId = (faustId: FaustId) =>
   `reservation-modal-${faustId}`;
 
 type ReservationModalProps = {
-  manifestation: Manifestation;
+  mainManifestation: Manifestation;
+  parallelManifestations?: Manifestation[];
 };
 
-const ReservationModal = ({
-  manifestation: {
-    pid,
-    materialTypes,
-    creators,
-    titles,
-    publicationYear,
-    edition
-  }
+const ReservationModalBody = ({
+  mainManifestation,
+  mainManifestation: { pid, materialTypes, titles, edition },
+  parallelManifestations
 }: ReservationModalProps) => {
+  const mainManifestationType = getManifestationType(mainManifestation);
+  const { availableManifestations } = UseAvailableManifestations({
+    manifestations: parallelManifestations ?? [mainManifestation],
+    type: mainManifestationType
+  });
```

Comment:
Please consider if filtering the manifestations by availability is actually what we want.

As I see it: An available manifestation is a manifestation where there is at least one holding at a library which can be picked up by a patron. However when we create a parallel reservation we should not limit ourselves to available manifestations. If we do so then patrons cannot reserve manifestations where all the holdings are out on loan already.

I tried to go through the specs but could not find any mention of this myself.

Reply:
Replaced with reservable and refactored...